README.md

```diff
@@ -10,23 +10,27 @@
   "description": "XDV Protocol metadata schema",
   "type": "object",
   "properties": {
-      "name": {
-          "type": "string",
-          "description": "Identifies the asset to which this token represents",
-      },
-      "description": {
-          "type": "string",
-          "description": "Describes the asset to which this token represents",
-      },
-      "image": {
-          "type": "string",
-          "description": "A URI pointing to a resource with mime type image/* representing the asset to which this token represents.",
-      };
-      "links": {
-          "type": "array",
-          "description": "A set of ipld links",
-      };
-  };
+    "name": {
+      "type": "string",
+      "description": "Identifies the asset to which this token represents",
+    },
+    "description": {
+      "type": "string",
+      "description": "Describes the asset to which this token represents",
+    },
+    "image": {
+      "type": "string",
+      "description": "A URI pointing to a resource with mime type image/* representing the asset to which this token represents.",
+    },
+    "links": {
+      "type": "array",
+      "description": "A set of ipld links",
+    },
+    "service": {
```

Comment:
no need to do this anymore, add https://docs.rs/multiformats/0.1.0/multiformats/ , we'll use it later